workflows/prognostic_c48_run/runtime/loop.py

```diff
@@ -390,6 +406,10 @@ def _compute_postphysics(self) -> Diagnostics:
             (self._tendencies, diagnostics, state_updates,) = self._postphysics_stepper(
                 self._state.time, self._state
             )
+            state_updates = _replace_precip_rate_with_accumlation(
```

Comment:
@brianhenn Can I get a quick check from you on this part of the PR? Just to make sure that correct update for 
 prescribed or ML-predicted surface precipitation is the accumulated precip as calculated here.